src/php/Api/Transfer/PhelFunction.php

```diff
@@ -85,16 +86,20 @@ public function doc(): string
 
     /**
      * @deprecated in favor of the signature attribute
+     *
+     * @return list<string>
      */
-    public function fnSignature(): string
+    public function fnSignature(): array
     {
         return $this->signature;
     }
 
     /**
      * @deprecated in favor of the signature attribute
```

Comment:
delete this deprecated notice, as this will be the method to access the first signature from now on